packages/webapp/src/api/models.ts

```diff
@@ -10,6 +10,32 @@ export const enum RetrievalMode {
   Text = 'text',
 }
 
+export type CustomStylesState = {
```

Comment:
this should probably not be in the api folder. Probably should be in components folder.